lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../enums/ds_align.enum.dart';
+import '../../models/ds_message_bubble_style.model.dart';
+import '../../themes/colors/ds_colors.theme.dart';
+import '../animations/ds_spinner_loading.widget.dart';
+import '../texts/ds_body_text.widget.dart';
+import '../utils/ds_cached_network_image_view.widget.dart';
+import 'ds_message_bubble.widget.dart';
+
+class DSLocationMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final DSMessageBubbleStyle style;
+  final String? title;
+  final String latitude;
+  final String longitude;
+
+  DSLocationMessageBubble({
+    super.key,
+    required this.align,
+    required this.latitude,
+    required this.longitude,
+    DSMessageBubbleStyle? style,
+    this.title,
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  final appKey = 'AIzaSyAlC3a3DZZBscR0QIbQpee13Op9Y05m_wc';
+
+  @override
+  Widget build(BuildContext context) {
+    final foregroundColor = style.isLightBubbleBackground(align)
+        ? DSColors.neutralDarkCity
+        : DSColors.neutralLightSnow;
+
+    return GestureDetector(
+      onTap: () => launchUrl(
+        Uri.parse(
+          'https://www.google.com/maps/search/?api=1&query=$latitude,$longitude',
```

Comment:
What if we use a simple package to open the coordinates on installed maps applications? I would like to suggest this package [map launcher](https://pub.dev/packages/map_launcher)

Sample code:

![Screenshot 2023-07-14 at 11 49 40](https://github.com/takenet/blip-ds-flutter/assets/20003773/eb82de7c-3251-407e-9d2c-13932f2940c9)